Add FAQ about coin requeue (#351)
zkSNACKs/WasabiDoc: docs/FAQ/FAQ-UseWasabi.md

If you want more anonymity you should CoinJoin multiple times.
:::

:::details
### Are coins automatically requeued after the round is complete?

> Yes, they are requeued based on your anonset setting.

Reviewer: ```suggestion
Yes they are, based on the anonymity set target.
```

> If it’s below the specified target then it’s queued automatically for another round.

Reviewer: ```suggestion
If the coins' anonymity sets are below the specified target after a round is complete, then the coins are requeued automatically for the next round.
```
